packages/Php/tests/Rector/FunctionLike/AddDefaultValueForUndefinedVariableRector/Fixture/take_static_into_account.php.inc

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Rector\Php\Tests\Rector\FunctionLike\AddDefaultValueForUndefinedVariableRector\Fixture;
+
+class WithAStaticVariableInsideMethod
+{
+    public function run()
+    {
+        static $variable = null;
+        if (null === $variable) {
+            $variable = 'not null anymore';
+        }
+
+        return $variable;
+    }
+}
+
+?>
```

Comment:
If the code should remain untoched, you can remove the part from there onwards (included)

Reply:
Do you intend all the code after `-----`?

Reply:
Nono, from this line included

Reply:
I've merge it, but see my commit